lib/modules/filteReddit.js

```diff
@@ -746,6 +755,17 @@ module.go = function() {
 			toggleNsfwFilter(toggle, true);
 		}
 	);
+
+	if (isPageType('linklist')) {
+		if (module.options.showRemoveVisitedTab.value && process.env.BUILD_TARGET !== 'safari') {
+			CreateElement.tabMenuItem({
```

Comment:
I like the idea of a line specifically for filter toggles. Add-on feature: opting to show custom toggles on that line.